setup.cfg

```diff
@@ -67,7 +67,7 @@ install_requires =
     dvc-task==0.1.2
     dvclive>=0.10.0
     dvc-data==0.10.1
-    dvc-http==2.19.1
+    dvc-http==2.27.2
```

Comment:
oh, btw, there seems to be a confusion regarding plugin naming. They started with the version of dvc in which they were detached, but then they should stick to a normal detached versioning scheme. Trying to keep up with dvc versioning is not that bad, but just saying that it is not strictly required.